broker-util/oo-admin-move

```diff
@@ -83,44 +88,54 @@ if timeout
   Rails.configuration.msg_broker[:rpc_options][:timeout] = timeout.to_i
 end
 
-app = nil
-user = nil
-gear = nil
-
-app, gear = Application.find_by_gear_uuid(gear_uuid)
-
-if gear.nil? or app.nil?
-  puts "ERROR: Gear not found: #{gear_uuid}"
-  exit 1
-end
-
-if app.scalable and node_profile
-  puts "Cannot specify 'node_profile' for a gear belonging to a scalable application."
-  exit 1
-end
-
-url = "http://#{app.name}-#{app.domain.namespace}.#{Rails.configuration.openshift[:domain_suffix]}"
-
-puts "URL: #{url}"
-puts "Login: #{app.domain.owner.login}"
-puts "App UUID: #{app._id}"
-puts "Gear UUID: #{gear._id}"
-
-destination_container = nil
-destination_container = OpenShift::ApplicationContainerProxy.instance(target_server_identity) if target_server_identity
-
-reply = nil
-begin
-  reply = gear.get_proxy.move_gear_secure(gear, destination_container, destination_district_uuid, change_district, change_region, node_profile)
-rescue OpenShift::NodeException => ne
-  puts ne.message
-  exit 1
-rescue OpenShift::UserException => ue
-  puts ue.message
-  exit 1
-end
-
-unless reply.nil?
-  puts "################# ADDITIONAL DEBUG OUTPUT #################\n#{reply.debugIO.string}\n" unless reply.debugIO.string.empty?
-  puts "################# ADDITIONAL ERROR OUTPUT #################\n#{reply.errorIO.string}\n" unless reply.errorIO.string.empty?
+move_status = {}
+gear_uuid.split(',').each do |uuid|
+    app = nil
+    user = nil
+    gear = nil
+
+    app, gear = Application.find_by_gear_uuid(uuid)
+
+    if gear.nil? or app.nil?
+      puts "ERROR: Gear not found: #{uuid}"
+      exit 1
+    end
+
+    if app.scalable and node_profile
+      puts "Cannot specify 'node_profile' for a gear belonging to a scalable application."
+      exit 1
+    end
+
+    url = "http://#{app.name}-#{app.domain.namespace}.#{Rails.configuration.openshift[:domain_suffix]}"
+
+    puts "URL: #{url}" unless json
+    puts "Login: #{app.domain.owner.login}" unless json
+    puts "App UUID: #{app._id}" unless json
+    puts "Gear UUID: #{gear._id}" unless json
+
+    destination_container = nil
+    destination_container = OpenShift::ApplicationContainerProxy.instance(target_server_identity) if target_server_identity
+    move_status[uuid] = { :login => app.domain.owner.login, :application => app._id, :destination => destination_container } if json
+
+    reply = nil
+    begin
+      gear.get_proxy.disable_log_debug! if json
+      reply = gear.get_proxy.move_gear_secure(gear, destination_container, destination_district_uuid, change_district, change_region, node_profile)
+      move_status[uuid].merge!({ :result => true })
+    rescue OpenShift::NodeException => ne
+      puts ne.message unless json
```

Comment:
do not overuse the 'statement condition' syntax as it makes the code hard to read.